assets/docs/pages/agentgateway/monitoring-agentgateway-nacks.md

```diff
@@ -0,0 +1,93 @@
+### About NACKs {#nacks}
+
+A NACK (negative acknowledgement) occurs when an {{< reuse "docs/snippets/agentgateway.md" >}} proxy rejects a configuration update from the {{< reuse "/docs/snippets/kgateway.md" >}} control plane. NACKs typically indicate configuration issues that prevent the proxy from applying the desired routing, policy, or backend settings. Typically the {{< reuse "/docs/snippets/kgateway.md" >}} control plane reports errors during translation, but some errors can only be caught at configuration time.
+
+{{< reuse "/docs/snippets/kgateway.md" >}} provides a combination of metrics and events to monitor the health of config synchronization between the {{< reuse "/docs/snippets/kgateway.md" >}} control plane and {{< reuse "docs/snippets/agentgateway.md" >}} proxies. The control plane exposes these metrics and events.
+
+For more general information about {{< reuse "/docs/snippets/kgateway.md" >}} control plane metrics, see [Control plane metrics](../../observability/control-plane-metrics/).
+
+Common causes of NACKs include:
+
+* CEL expressions in policies that pass the control plane validator but the data plane rejects (the control plane uses a Go-based CEL validator, while agentgateway uses a Rust-based validator, which can have different validation behavior)
+* Invalid TLS certificates
+
+When a NACK occurs, {{< reuse "/docs/snippets/kgateway.md" >}} provides two observability signals:
+
+* **Prometheus metric**: A counter metric tracks the total number of NACKs across all {{< reuse "docs/snippets/agentgateway.md" >}} proxies which kgateway manages
+* **Kubernetes Events**: {{< reuse "/docs/snippets/kgateway.md" >}} creates Warning events on both the `Gateway` and its corresponding `Deployment` with details about the error
+
+### Monitor NACKs with control plane metrics {#nack-metrics}
+
+You can access the {{< reuse "/docs/snippets/kgateway.md" >}} control plane metrics endpoint to view NACK metrics that track configuration rejections from {{< reuse "docs/snippets/agentgateway.md" >}} proxies.
+
+#### View the NACK metric {#view-nack-metric}
+
+1. Port-forward the control plane deployment on port 9092.
+   ```sh
+   kubectl -n {{< reuse "docs/snippets/namespace.md" >}} port-forward deployment/{{< reuse "/docs/snippets/helm-kgateway.md" >}} 9092
+   ```
+
+2. Open the {{< reuse "/docs/snippets/kgateway.md" >}} control plane [metrics endpoint](http://localhost:9092/metrics).
+
+3. Search for the `kgateway_agentgateway_xds_rejects_total` metric.
+
+   {{< callout type="info" >}}
+   The metric will only appear after at least one NACK has been reported. If no NACKs have occurred, the metric will not be present in the metrics output.
```

Comment:
totally not necessary, but is it possible to provide a dummy resource that would cause a nack? in case they want to try it out for themselves?